redash/query_runner/databricks.py

```diff
@@ -35,6 +36,53 @@ def _build_odbc_connection_string(**kwargs):
     return ";".join([f"{k}={v}" for k, v in kwargs.items()])
 
 
+def split_sql_statements(query):
```

Comment:
Since this is the function that works for all sql statement, Is it better to place it into the BaseSQLQueryRunner?

Reply:
It's not needed for all query runners, a lot of them support multiple sql statements on their own (e.g. PostgreSQL-based, MySQL/MariaDB, etc.). Also, this function does some extra job needed only for Databricks - it cannot handle comments after semicolon, so we have to strip them (but keep any garbage so user will get a proper error message). This logic most likely will be irrelevant for any other query runner